docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -148,6 +151,8 @@ Non-Goals listed in this document are intended to scope bound the current v1alph
 8. As a cluster operator, I want to be able to quickly respond to a non-Kubernetes CVE that affects my base image or Kubernetes dependencies by upgrading my clusters in an automated fashion.
 9. As a cluster operator, I would like to upgrade to a new minor version of Kubernetes so that my cluster remains supported.
 10. As a cluster operator, I want to know that my cluster isn’t working properly after creation. I have ended up with an API server I can access, but kube-proxy isn’t functional or new machines are not registering themselves with the control plane.
+11. As a cluster operator I would like to use appropriate rollout strategy to scale down my control plane during upgrade.
+12. As a cluster operator I want to be able to scale up my control plane during upgrade.
```

Comment:
These use case probably need some more details, scale up/down are normal operations that happen during a rollout, while in practice we want to allow users to change how the order operation is performed